### DISCUSSION TITLE
Reader ROMS not reading lon_u, lat_u, lon_v and lat_v?

### DISCUSSION THREAD

**Discussion by AdriSanj** [OpenDrift/opendrift#1430] [Q&A]
Dear Knut Frode,

These days in our working group we have detected a displacement in the graphical representation of the velocity fields with respect to the land_mask using a ROMS model, which we read from the reader_ROMS_native.py of the OpenDrift v1.11.12 version. The ROMS files we are working with include the velocities in the u and v points (i.e. they have lon_u  lat_u for u points and lon_v and lat_v for v points). From this, we started to check what this could be due to, and we believe that the velocities of points u and v of the ROMS model are read in such a way that it places them in the rho points (as shown in the get_variables function of the reader_ROMS_native.py). Is this so? If so, will an adjustment be made in the code so that the velocities can be read at the corresponding u and v points? Thanks in advance for your answer.

Regards.

**Comment by awesome-geek-ripiv** [comment]
this appears to address the same issue:

https://github.com/OpenDrift/opendrift/discussions/1315#discussioncomment-11088394
